### PR TITLE
Add BZ#2023764 to known issues for MTV 2.2 Release notes

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.2

Resolves https://bugzilla.redhat.com/show_bug.cgi?id=2032014, which refers to BZ 2023764.

Preview: last item in https://deploy-preview-277--forklift-documentation.netlify.app/documentation/doc-release_notes/master/#known-issues-22_forklift (Immediately before the "Forklift 2.1" header)

### CODE REVIEW

File: documentation/modules/rn-2.2.adoc [hunk 1]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.Log archive files downloaded from the {project-short} web console might include logs related to a deleted migration plan
```
Comment (mguetta1) [inline documentation/modules/rn-2.2.adoc:98]: 1. "Log archive files" -> it is one file that includes several logs
2. Please change the end to "deleted resource" - can be deleted plan or target VM resource

File: documentation/modules/rn-2.2.adoc [hunk 2]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
...
+
+If you do either of the following in the {project-short} web console, the log archive you download might contain logs related to a deleted migration plan:
```
Comment (mguetta1) [inline documentation/modules/rn-2.2.adoc:100]: 1. Please remove "in the {project-short} web console" - the user can delete the resources via CLI
2. and mention that: the "log archive you download **from the {project-short} web console**
3. Please add that "Or deleted target VM" at the end

File: documentation/modules/rn-2.2.adoc [hunk 3]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
...
+
+* Delete a migration plan and then recreate the plan using the same name
+* Delete a migrated VM from the destination side and then re-migrate it using a new migration plan
+
+In either case, the *Get logs* button, which is used to collect the logs of a new migration, does not appear following the migration as expected. Instead, the *Download logs* button, which is used to download archived logs, is immediately enabled. As a result, the downloaded logs might include logs from a previous job involving a deleted migration plan or a VM with the same name (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764#c1[BZ#2023764]).
```
Comment (mguetta1) [inline documentation/modules/rn-2.2.adoc:105]: The attached link redirects to the first comment. could you please change it to be the bug main page?

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/rn-2.2.adoc [hunk 4]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
...
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.A log archive file downloaded from the {project-short} web console might include logs related to a deleted resource
+
+If you do either of the following, the log archive file you download from the {project-short} web console might contain logs related to a deleted migration plan or a deleted target VM:
+
+* Delete a migration plan and then recreate the plan using the same name
+* Delete a migrated VM from the destination side and then re-migrate it using a new migration plan
+
+In either case, the *Get logs* button, which is used to collect the logs of a new migration, does not appear following the migration as expected. Instead, the *Download logs* button, which is used to download archived logs, is immediately enabled. As a result, the downloaded logs might include logs from a previous job involving a deleted resource with the same name (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[BZ#2023764]).
```
Comment (mguetta1) [inline documentation/modules/rn-2.2.adoc:105]: Looks like the BZ link should be bold (compared to others)

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Review by mguetta1 — APPROVED** [review]
Looks good to me

File: documentation/modules/rn-2.2.adoc [hunk 5]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.A log archive file downloaded from the {project-short} web console might include logs related to a deleted resource
```
Comment (apinnick) [inline documentation/modules/rn-2.2.adoc:98]: This looks like a copy/paste directly from the bug. Please rewrite according to our style, for example, briefer title.

File: documentation/modules/rn-2.2.adoc [hunk 6]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
...
+
+If you do either of the following, the log archive file you download from the {project-short} web console might contain logs related to a deleted migration plan or a deleted target VM:
```
Comment (apinnick) [inline documentation/modules/rn-2.2.adoc:100]: Rewrite the first clause. "either of the following" is not clear style.

File: documentation/modules/rn-2.2.adoc [hunk 7]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
...
+
+* Delete a migration plan and then recreate the plan using the same name
```
Comment (apinnick) [inline documentation/modules/rn-2.2.adoc:102]: Make these two bullet points  more concise.

File: documentation/modules/rn-2.2.adoc [hunk 8]
```diff
@@ -94,3 +94,12 @@ The error status message for a VM with no operating system on the *Migration pla
...
+* Delete a migrated VM from the destination side and then re-migrate it using a new migration plan
+
+In either case, the *Get logs* button, which is used to collect the logs of a new migration, does not appear following the migration as expected. Instead, the *Download logs* button, which is used to download archived logs, is immediately enabled. As a result, the downloaded logs might include logs from a previous job involving a deleted resource with the same name (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*]).
```
Comment (apinnick) [inline documentation/modules/rn-2.2.adoc:105]: Avoid "in either case". Ambiguous and usually unneccessary.

**Review by apinnick — CHANGES_REQUESTED** [review]
Please make this known issue shorter and more concise.

File: documentation/modules/rn-2.2.adoc [hunk 9]
```diff
@@ -94,3 +94,9 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.A downloaded log archive file might include logs from a deleted resource
```
Comment (apinnick) [inline documentation/modules/rn-2.2.adoc:98]: Change to "Log archive file includes logs of a deleted migration plan or VM"

You don't need "might" in the header. You also don't need "downloaded". That can be explained in the description.

File: documentation/modules/rn-2.2.adoc [hunk 10]
```diff
@@ -94,3 +94,9 @@ The error status message for a VM with no operating system on the *Migration pla
...
+
+A log archive file downloaded from the {project-short} web console might include logs from a deleted resource if it was generated after you deleted a migration plan and recreated it with the same name _or_ after you deleted a migrated VM from the destination side and re-migrated the VM,
```
Comment (apinnick) [inline documentation/modules/rn-2.2.adoc:100]: Suggest this:

If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. [bug link]

**Review by apinnick — CHANGES_REQUESTED** [review]
Added comments.

File: documentation/modules/rn-2.2.adoc [hunk 11]
```diff
@@ -94,3 +94,7 @@ The error status message for a VM with no operating system on the *Migration pla
 .Network, storage, and VM referenced by name in the Plan CR are not displayed in the web console.
 
 If a Plan CR references storage, network, or VMs by name instead of by ID, the resources do not appear in the {project-short} web console. The migration plan cannot be edited or duplicated. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986020[*BZ#1986020*])
+
+.Log archive file includes logs of a deleted migration plan or VM
+
+If you delete a migration plan and then run a new migration plan with the same name or if you delete a migrated VM and then remigrate the source VM, the log archive file created by the {project-short} web console might include the logs of the deleted migration plan or VM. [bug link] (link:https://bugzilla.redhat.com/show_bug.cgi?id=2023764[*BZ#2023764*]).
```
Comment (mguetta1) [inline documentation/modules/rn-2.2.adoc:100]: [bug link] should be removed

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Review by mguetta1 — APPROVED** [review]
Looks good to me. Thanks

**Review by apinnick — APPROVED** [review]

**Pull request merged**